policy/policy.sample.json

```diff
@@ -0,0 +1,78 @@
+{
+    "SecurityPolicies": [
+        {
+            "Name": "policy1",
+            "PolicyID": "3710940D-FBC0-4A44-A95D-E549E628B4DF", # unique identifier across tenants, auto generated for POST.
+            "AppliedTo": [ # can attach multiple tenants to which the policy can be applied to.
+                {
+                    "Tenant":"tenant1", #optional if TenantID below is provided 
+                    "TenantID": "3A6E467F-370B-43EA-BCEC-C28912EDD752", # optional but good to have to relate things as names are not unique
+                    "Segment": "Segment1", # optional
+                    "SegmentID": "01CA06A0-429B-4EE1-9185-F17D5CE4D0F2",
+                },
+                {
+                    "Tenant":"tenant1",
+                    "Segment": "Segment2",
+                },
+            ],
+            "Tags": [
+                {
+                    "Key": "Platform",
+                    "Value": "Openstack",
+                },
+                {
+                    "Key": "TenantID",
+                    "Value": "D46FD18B-A381-4297-A6D4-9DA594EE71C8",
+                },
+            ], # meta data attached to policies for various external enviorments like openstack/kybernetes
+            "Direction" : "Egress", # can be Egress or Ingress.
+            "Peers": [
```

Comment:
"The other side". In an ingress rule it would be the sender of the traffic, in an egress rule it would be the receiver of the traffic.

There is something I don't like about the word "peer", just wasn't able to come up with something better, yet.

The things is that "AppliedTo" and "Peer" are the two ends of the connection, but which one is sender and receiver changes depending on whether it's an ingress or egress rule. It seems though that we should have more similar words than "AppliedTo" and "Peer".

"AppliedTo" is a good name, though, since it says very clearly what's going to happen: The rule is going to be implemented on the hosts where those endpoints are located.